docker-compose.yml

```diff
@@ -133,6 +133,12 @@ services:
     volumes:
     - ./oaipmh/fiz-oai-backend.properties:/usr/local/tomcat/conf/fiz-oai-backend.properties:ro
     - backend-logs:/usr/local/tomcat/logs
+    labels:
+      - traefik.http.routers.oai-provider.rule=Host(`oai.portal.mardi4nfdi.${TLD}`)
+      - traefik.http.routers.oai-provider.entrypoints=websecure
+      - traefik.http.routers.oai-provider.tls.certResolver=le
+      - traefik.http.middlewares.test-auth.basicauth.users=bWFyZGktdGVzdC11c2VyOnRlc3Q=
+
```

Comment:
one needs to connect the auth middleware with the router, e.g,

traefik.http.routers.service-${service-name}.middlewares=${auth-name}

Reply:
Should I add this line ?

traefik.http.routers.service-${oai-input.portal.mardi4nfdi.${TLD}}.middlewares=${oai-auth}

Reply:
I get the feeling that you don't fully understand what a traefik middelware is, see https://doc.traefik.io/traefik/middlewares/overview/